pkg/config/apm.go

```diff
@@ -110,10 +110,26 @@ func setupAPM(config Config) {
 	})
 
 	config.SetEnvKeyTransformer("apm_config.filter_tags.require", func(in string) interface{} {
+		if len(in) > 0 && in[0] == '[' {
+			var values []string
```

Comment:
I think the code would have less indentation overall if you'd reverse the condition and return with the `strings.Split`. There'd be 1 line indented vs 6.